### PR TITLE
LPS-95079

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
please publish `project-templates` Tests passed here: https://github.com/lawrence-lee/liferay-portal/pull/111

File: modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle [hunk 1]
```diff
@@ -31,10 +31,15 @@ dependencies {
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "4.4.0"
 #end
 	compileOnly group: "com.liferay.portal", name: "com.liferay.util.bridges", version: "2.0.0"
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle]: hey @lawrence-lee  I think we need to do the same thing for `com.liferay.util.bridges` update the templates with the 7.0.x, 7.1.x and 7.2.x versions.  in the future, this is going to go away with improved TP plugin.

Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates-freemarker-portlet/src/main/resources/archetype-resources/build.gradle]: Closing PR. will resend with versions from the BOMs

File: modules/build.gradle [hunk 2]
```diff
@@ -175,6 +176,17 @@ gradle.beforeProject {
 
 				rename "gitignore", ".gitignore"
 			}
+
+            jar {
+                doFirst {
+                    DirectoryScanner.defaultExcludes.each { DirectoryScanner.removeDefaultExclude it }
```
[the remark this answers is not in the document]
Comment (gamerson) [reply modules/build.gradle]: like
```
jar {
                doFirst {
                    DirectoryScanner.removeDefaultExclude SelectorUtils.DEEP_TREE_MATCH + "/.gitignore"
                }

                 doLast {
                    DirectoryScanner.resetDefaultExcludes()
                }
            }
```

**Comment by lawrence-lee** [comment]
sent here: https://github.com/petershin/liferay-portal/pull/518

**Pull request closed without merging**